File: 1960-CheckIfTheSentenceIsPangram/1960-CheckIfTheSentenceIsPangram.py
# Last updated: 8/5/2025, 2:52:43 PM
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        if len(sentence) < 26:
            return False
        characters = set()
        for i in sentence:
            characters.add(i)
        if len(characters) == 26:
            return True
        return False
        

File: 1960-CheckIfTheSentenceIsPangram/1960-CheckIfTheSentenceIsPangram.java
// Last updated: 8/5/2025, 2:52:46 PM
class Solution {
    public boolean checkIfPangram(String sentence) {
        HashSet<Character> set = new HashSet<>();
        for(Character c : sentence.toLowerCase().toCharArray())
            set.add(c);
        
        return set.size() == 26;
    }
}